Tests/Tracking/CustomerIOTest.swift

```diff
@@ -21,7 +21,7 @@ class CustomerIOTest: UnitTest {
     }
 
     func test_initialize_expectAddModuleHooks_expectRunCleanup() {
-        customerIO.postInitialize(siteId: testSiteId, diGraph: diGraph)
+        customerIO.postInitialize(diGraph: diGraph)
```

Comment:
I couldn't find a test case, that checks that the `siteID` and `apiKey` are correctly inserted and retrieved from `Config`

Reply:
Added https://github.com/customerio/customerio-ios/pull/300/commits/b3cad3af618b04742fb9fb9be0b725487fc5a563